edgelet/Cargo.lock

```diff
@@ -1496,6 +1490,11 @@ dependencies = [
  "winapi 0.3.5 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```

Comment:
Needs to be added to `THIRDPARTYNOTICES`

Reply:
Done